File: class21/java/AP2109-ContaTentativas.java
public class ContaTentativas {
    public static void main(String[] x) {
        int secreto = 42; int[] palpites = {50, 25, 40, 42}; int tent = 0;
        for (int p : palpites) { tent++; if (p == secreto) break; }
        System.out.println("Acertou em " + tent + " tentativas");
    }
}

File: class21/cpp/AP2109-ContaTentativas.cpp
#include <iostream>
int main(){int secreto=42,palpites[4]={50,25,40,42},tent=0;for(int i=0;i<4;i++){tent++;if(palpites[i]==secreto)break;}std::cout<<"Acertou em "<<tent<<" tentativas\n";return 0;}
